### DISCUSSION TITLE
Compilation speed

### DISCUSSION THREAD

**Discussion by erfty125** [ZhymabekRoman/Exagear-For-Termux#17] [Q&A] [answered]
I want to compile hldsk-portable with ndk but i have to use exatermux , so the question is: " compiling on exagear is faster than on exatermux? "

**Comment by ZhymabekRoman** [comment, marked as answer]
> I want to compile hldsk-portable with ndk but i have to use exatermux , so the question is: " compiling on exagear is faster than on exatermux? "

If you mean Exagear Windows vs ExagearTermux, Exagear Windows is faster but there is a problems that makes you not be able to compile properly. So use Exagear Termux

**Reply by erfty125** [reply]
Ok , thank you
